compiler/testData/integration/ant/jvm/kt16227/build.xml

```diff
@@ -0,0 +1,8 @@
+<project name="Ant Task Test" default="build">
+    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>
+    <target name="build">
+        <kotlinc src="${test.data}" output="${temp}/test.jar">
+        </kotlinc>
+
+    </target>
+</project>
```

Comment:
Looks like this test doesn't check anything that isn't already checked by other tests.

I suppose we need a test that somehow checks that with the jar output, no `includeRuntime` leads to the jar file being created without stdlib classes in it.

And also, another test that does use the `includeRuntime` option of the Ant plugin (currently there are zero tests on it) and checks that the resulting jar file has the stdlib classes.

Reply:
A new test has been added, which considered three different situations(includeRuntime=true/false/not specified).